addons/survey/views/survey_survey_views.xml

```diff
@@ -74,11 +74,14 @@
                                     <field name="is_page" invisible="1"/>
                                     <field name="questions_selection" invisible="1"/>
                                     <field name="survey_id" invisible="1"/>
-                                    <field name="triggering_question_id" invisible="1"/>
                                     <field name="random_questions_count"
                                         attrs="{'column_invisible': [('parent.questions_selection', '=', 'all')], 'invisible': [('is_page', '=', False)]}"/>
-                                    <button disabled="disabled" icon="fa-code-fork" attrs="{'invisible': [('triggering_question_id', '=', False)]}"
-                                        title="This question depends on another question's answer." class="icon_rotates"/>
+                                    <field name="triggering_question_id" widget="trigger" nolabel="1"/>
+                                    <field name="triggering_answer_id" invisible="1"/>
+                                    <!--add to fieldNames to force sync when opening record (support reordering/deletions)-->
```

Comment:
"add to fieldNames" ?

I think I understand the purpose, but the explanation is a bit too technical.